src/app/auth/login/login.component.html

```diff
@@ -1,2 +1,8 @@
-<p>login works!</p>
-<button (click)="handleClick()">Go next page</button>
+<h2>Runners</h2>
+<form action="#">
```

Comment:
Concern forms, you will need to learn how it works in angular. In this case you need to use or template driven forms or reactive forms. I prefer use reactive forms. Learn please this chapter https://angular.io/guide/forms-overview